src/main/java/com/autotune/analyzer/utils/AnalyzerConstants.java

```diff
@@ -551,6 +551,11 @@ private MetricNameConstants() {
     public static final class RecommendationNotificationMsgConstant {
         public static final String NOT_ENOUGH_DATA = "There is not enough data available to generate a recommendation.";
         public static final String DURATION_BASED_AVAILABLE = "Duration Based Recommendations Available";
+        public static final String CPU_REQUEST_NOT_SET = "CPU Request was not set";
+        public static final String MEMORY_REQUEST_NOT_SET = "Memory Request was not set";
+        public static final String MEMORY_LIMIT_NOT_SET = "Memory Limit was not set";
+        public static final String CPU_UNDER_PROVISIONED = "Current CPU requests are under-provisioned, Please update the requests with recommended value";
+        public static final String MEMORY_UNDER_PROVISIONED = "Current Memory requests are under-provisioned, Please update the requests with recommended value";
```

Comment:
Until we can test with actual data and are sure of the algo, dont want to have these notifications as yet. Let us limit the initial set to checking for requests (both CPU and Mem) and limits (Memory only) not being set

Reply:
@dinogun Thanks for your review, Even I was thinking about that so added it as a separate commit, Here are some of my views

I can revert to older commit and remove this change completely or can comment this code and add a comment saying should be uncommented after the algos are set. Which makes us not to loose track of chnages to be done after algos. Please lemme know how we could proceed? I'm more inclined to comment the code but not remove it if you are comfortable